### PR TITLE
Support Python 3.10

### PR DESCRIPTION
**Pull request by SagiPolaczek**
Adding support to `python 3.10`.
NOTE that the CI python environment is set to 3.10 .

### CODE REVIEW

File: run_all_unit_tests.sh [hunk 1]
```diff
@@ -30,7 +30,7 @@ create_env() {
         requirements+=$(cat examples/requirements.txt)
     fi
 
-    PYTHON_VER=3.7
+    PYTHON_VER=3.10
```
Comment (mosheraboh) [inline run_all_unit_tests.sh:33]: Now that we see that it pass - you can change it to 3.8

File: Jenkinsfile [hunk 2]
```diff
@@ -13,7 +13,7 @@ pipeline {
             steps {
                 sh '''
                 rm -rf test-reports/TEST-*.xml
-                jbsub -wait -out ccc_log.txt -queue x86_1h -mem 40g -cores "4+1" ./run_all_unit_tests.sh 11.6 /dccstor/fuse_med_ml/cicd/envs/
+                jbsub -wait -out ccc_log.txt -queue x86_1h -mem 40g -cores "10+1" ./run_all_unit_tests.sh 11.6 /dccstor/fuse_med_ml/cicd/envs/
```
Comment (mosheraboh) [inline Jenkinsfile:16]: We though maybe to add also:
-require v100

**Review by mosheraboh — APPROVED** [review]
Great, thanks!
Can you please also update the readme file and remove "(3.7 is recommended)"?

File: Jenkinsfile (hunk 2 above)
Comment (SagiPolaczek) [reply Jenkinsfile:16]: Correct.
I accidentally added `-require *a100*` to ISIC's PR, so I'm updating both now.

**Review by mosheraboh — APPROVED** [review]
Looks good!

**Pull request merged**